packages/taquito-michel-codec/src/base58.ts

```diff
@@ -91,8 +91,8 @@ export class Base58DecodingError extends Error {
  *  @category Error
  *  @description
  */
-export class InvalidMessageError extends Error {
-  public name = 'InvalidMessageError';
+export class InvalidHashLengthError extends Error {
```

Comment:
might overlook this the last time, why do we want to change class name from `InvalidMessageError` to `InvalidHashLengthError`?